pkg/kubectl/cmd/cmd_test.go

```diff
@@ -416,6 +416,59 @@ func ExamplePrintReplicationControllerWithWide() {
 							Name:  "foo",
 							Image: "someimage",
 						},
+						{
+							Name:  "foo2",
+							Image: "someimage",
+						},
+					},
+				},
+			},
+		},
+		Status: api.ReplicationControllerStatus{
+			Replicas: 1,
+		},
+	}
+	err := f.PrintObject(cmd, ctrl, os.Stdout)
+	if err != nil {
+		fmt.Printf("Unexpected error: %v", err)
+	}
+	// Output:
+	// NAME      DESIRED   CURRENT     AGE       CONTAINER(S)   IMAGE(S)    SELECTOR
+	// foo       1         1           10y       foo            someimage   foo=bar
+	//           foo2      someimage
```

Comment:
What I want is:
```
	// NAME      DESIRED   CURRENT     AGE       CONTAINER(S)   IMAGE(S)    SELECTOR
 	// foo       1         1           10y       foo,foo2            someimage,someimage2   foo=bar
```

1 line

No spaces in comma-separated lists, so columns can be `cut`.